### DISCUSSION TITLE
Cannot get Nx Console working in Rider

### DISCUSSION THREAD

**Discussion by jvskriubakken** [nrwl/nx-console#1638] [Q&A] [answered]
Hi,

the gui for the Nx Console does not appear at all after installation of the plugin. Has anyone tested it on Rider?

**Comment by Cammisuli** [comment]
It might not appear immediately. If you open the command prompt, and search for Nx Console, does it appear there so that it can open the view on the side panel?

**Reply by jvskriubakken** [reply]
The issue was that the version I tried did not yet have any UI window in Jetbrains/Rider

**Reply by jvskriubakken** [reply, marked as answer]
Now it works fine.
